Makefile

```diff
@@ -56,17 +58,20 @@ LDFLAG_OPTIONS = -ldflags "-X sigs.k8s.io/kubefed/pkg/version.version=$(GIT_VERS
 
 GO_BUILDCMD = CGO_ENABLED=0 go build $(VERBOSE_FLAG) $(LDFLAG_OPTIONS)
 
+GO_TESTCMD  = CGO_ENABLED=0 go test $(COMPILE_FLAG) $(LDFLAG_OPTIONS)
+
 TESTARGS ?= $(VERBOSE_FLAG) -timeout 60s
 TEST_PKGS ?= $(GOTARGET)/cmd/... $(GOTARGET)/pkg/...
 TEST_CMD = go test $(TESTARGS)
 TEST = $(TEST_CMD) $(TEST_PKGS)
 
 DOCKER_BUILD ?= $(DOCKER) run --rm -v $(DIR):$(BUILDMNT) -w $(BUILDMNT) $(BUILD_IMAGE) /bin/sh -c
+DOCKER_BINARY_BUILD ?= $(DOCKER) run --rm -v $(DIR):$(GOPATH)/src/$(GOTARGET) -w $(GOPATH)/src/$(GOTARGET) $(BUILD_IMAGE) /bin/sh -c
```

Comment:
Since both are building binaries how about renaming this to `DOCKER_GOPATH_BUILD` or similar?

Reply:
Fixed.